server/src-lib/Hasura/Server/CheckUpdates.hs

```diff
@@ -47,14 +47,9 @@ checkForUpdates (LoggerCtx loggerSet _ _) manager = do
   where
     updateMsg v = "Update: A new version is available: " <> v
     getUrl = do
-      let buildUrl a = "https://releases.hasura.io/graphql-engine?agent="
-                       <> a
-                       <> "&version="
-                       <> currentVersion
-      isCI <- lookupEnv "CI"
-      case isCI of
-        Just "true" -> return $ buildUrl "server-ci"
-        _           -> return $ buildUrl "server"
+      let buildUrl agent = "https://releases.hasura.io/graphql-engine?agent=" <>
+                           agent <> "&version=" <> currentVersion
+      buildUrl . bool "server" "server-ci" <$> CI.isCI
```

Comment:
@ajeetdsouza Let's add the CI name here also?

- `server` if there is no CI
- `server-ci-<ci-name>` if there is a CI

@dsandip what do you think?